### PR TITLE
[INLONG-8875][Release] Update changes log for the 1.9.0 version

### LINKED ISSUE #8875: [Release] Update changes log for the 1.9.0 version
### Description

_No response_

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by luchunliang**
### Prepare a Pull Request

- Fixes #8875 

### Motivation

Update changes log for the 1.9.0 version

### CODE REVIEW

**Review by gosonzhang — APPROVED** [review]

File: CHANGES.md [hunk 1]
```diff
@@ -22,229 +22,246 @@
...
+| [INLONG-8690](https://github.com/apache/inlong/issues/8690) | [Improve][Security] TemporaryFolder on unix-like systems does not limit access to created files type/improve      |
+| [INLONG-8688](https://github.com/apache/inlong/issues/8688) | [Improve][Security] Incorrect Authorization in MySQL Connector Java type/improve                                  |
+| [INLONG-8687](https://github.com/apache/inlong/issues/8687) | [Improve][Security] Apache Pulsar Java Client vulnerable to Improper Certificate Validation type/improve          |
+| [INLONG-8686](https://github.com/apache/inlong/issues/8686) | [Improve][Security] TemporaryFolder on unix-like systems does not limit access to created files type/improve      |
+| [INLONG-8685](https://github.com/apache/inlong/issues/8685) | [Improve][Security] Denial of service due to parser crash type/improve                                            |
+| [INLONG-8684](https://github.com/apache/inlong/issues/8684) | [Improve][Security] Vite Server Options (server.fs.deny) can be bypassed using double forward-slash (//) component/dashboard type/improve    |
```
Comment (vernedeng) [inline CHANGES.md:264]: plz re-format it

File: CHANGES.md [hunk 2]
```diff
@@ -22,229 +22,246 @@
...
-| [INLONG-8467](https://github.com/apache/inlong/issues/8467) | [Improve][Dashboard] Audit query condition optimization					                                         | 
+### TubeMQ
+|                            ISSUE                            | Summary                                                                                                           |
+|:-----------------------------------------------------------:|:------------------------------------------------------------------------------------------------------------------|
+| [INLONG-4972](https://github.com/apache/inlong/issues/4972) | [Feature][TubeMQ] Add TubeMQ Command Tools stage/roadmap type/feature                                             |
+| [INLONG-8903](https://github.com/apache/inlong/issues/8903) | [Bug][TubeMQ][Sort] int64 not recognized by the compiler and Missing @Override annotations type/bug               |
+| [INLONG-8871](https://github.com/apache/inlong/issues/8871) | [Improve][TubeMQ] Use an error code in checkMessageAndStatus() to return the check result instead of throwing an exception component/tubemq type/improve    |
```
Comment (vernedeng) [inline CHANGES.md:241]: ditto

File: CHANGES.md [hunk 3]
```diff
@@ -22,229 +22,246 @@
...
-| [INLONG-8339](https://github.com/apache/inlong/issues/8339) | [Improve][DataProxy][Agent] Enable audit by default in DataProxy and Agent | 
-| [INLONG-8347](https://github.com/apache/inlong/issues/8347) | [Improve][Agent] Optimize  the agent UT of testTimeOffset                  |
-| [INLONG-8352](https://github.com/apache/inlong/issues/8352) | [Improve][Agent] Optimize the agent UT of testRestartTriggerJobRestore     |
-| [INLONG-8376](https://github.com/apache/inlong/issues/8376) | [Improve][Agent] Optimize the agent UT of TestTriggerManager               |
+|                            ISSUE                            | Summary                                                                                                           |
+|:-----------------------------------------------------------:|:------------------------------------------------------------------------------------------------------------------|
+| [INLONG-8850](https://github.com/apache/inlong/issues/8850) | [Improve][Agent] Remove unregister of MetricRegister when taskmanager is initialized type/improve                 |
```
Comment (aloyszhang) [inline CHANGES.md:29]: Why add `type/improve` here?

File: CHANGES.md (hunk 1 above)
Comment (luchunliang) [reply CHANGES.md:264]: Confirm with vernedeng, it is ok

File: CHANGES.md (hunk 2 above)
Comment (luchunliang) [reply CHANGES.md:241]: Confirm with vernedeng, it is ok

**Review by vernedeng — APPROVED** [review]

**Review by aloyszhang — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**